products/opensuse/main.pm

```diff
@@ -344,7 +344,7 @@ sub load_reboot_tests() {
     if (check_var('ARCH', 's390x')) {
         loadtest "installation/reconnect_s390";
     }
-    if (installyaststep_is_applicable()) {
+    if (installyaststep_is_applicable() && !check_var('ARCH', 's390x')) {
```

Comment:
why should first_boot or boot_encrypt not apply to s390x? why not put it into installyaststep_is_applicable with a comment why it does not apply for s390x?